core/parentchain/indirect-calls-executor/src/indirect_calls_executor.rs

```diff
@@ -120,6 +125,21 @@ where
 				}
 			};
 
+			// Found Ack_Game extrinsic in block.
+			if let Ok(xt) =
+			UncheckedExtrinsicV4::<AckGameFn>::decode(&mut xt_opaque.encode().as_slice())
+			{
+				if xt.function.0 == [GAME_REGISTRY_MODULE, ACK_GAME] {
+					if let Err(e) = self.handle_ack_game_xt(&xt) {
+						error!("Error performing acknowledge game. Error: {:?}", e);
+					} else {
+						// Cache successfully executed shielding call.
+						executed_shielding_calls.push(hash_of(xt))
```

Comment:
I don't think that's right. That's not a shielding call, so you don't need to push here. Simply printing an info message should be enough here for now.

Reply:
Ah well, but it's an executed extrinsic... so leave it. We should rename to `executed_extrinsics`

Reply:
Leave as is, but rename `executed_shielding_calls` to `executed_extrinsics`